plugins/vocabbuilder.koplugin/main.lua

```diff
@@ -242,6 +242,133 @@ function MenuDialog:init()
             })
         end,
     }
+    local interval_settings_button = {
+        text = _("Interval settings"),
+        callback = function()
+            -- review intervals
+            settings.default_review_intervals = {5, 30, 720, 1440}
+            settings.default_review_intervals_pretty = "5m, 30m, 12h, 24h"
```

Comment:
Ah, true. This could instead be a local variable. Will see if that works.

Reply:
There's an issue with that. I use these values in both files, so that's done via the settings file. I'm not sure how to approach that otherwise.

Reply:
One naive way is to just use literals at both places, since the default is not supposed to change.
Another way is to store the default in `DB`(`VocabularyBuilder` in `db.lua`) so you can reference it from both files.

Reply:
Alright. For starters, let's use local variables. This would introduce duplication of the variable default_interval_modifier.